### PR TITLE
taxonomy details visible in work detail view

### PR DESCRIPTION
**Pull request by kaysiz**
user can see taxonomy details in work detail view - WorkOverviewView

### CODE REVIEW

File: indigo_app/templates/indigo_api/work_overview.html [hunk 1]
```diff
@@ -67,6 +67,17 @@ <h4 class="text-muted">{{ work.frbr_uri }}</h4>
                     <br>Commenced by <a href="{% url 'work' frbr_uri=work.commencing_work.frbr_uri %}" data-popup-url="{% url 'work_popup' frbr_uri=work.commencing_work.frbr_uri %}">{{ work.commencing_work.title }}</a>
                     {% endif %}
                   </dd>
+
+                  {% regroup work.taxonomies.all by taxonomy_vocabulary as topic_groups %}
+                  {% if topic_groups %}
+                      {% for topic in topic_groups %}
+                        <dt>{{ topic.grouper }}</dt>
+                          {% for taxonomy in topic.list %}
```
Comment (longhotsummer) [inline indigo_app/templates/indigo_api/work_overview.html]: Inside a `dl` element, there should only be `dt` and `dd` tags. So wrap the for loop in a `dd` tag.

**Comment by longhotsummer** [comment]
Screenshot please :)

File: indigo_app/templates/indigo_api/work_overview.html (hunk 1 above)
Comment (kaysiz) [reply indigo_app/templates/indigo_api/work_overview.html]: i have updated the file with the suggestions, however it looks like the line spaces are a bit big between the `dd` tags

File: indigo_app/templates/indigo_api/work_overview.html [hunk 2]
```diff
@@ -67,6 +67,17 @@ <h4 class="text-muted">{{ work.frbr_uri }}</h4>
                     <br>Commenced by <a href="{% url 'work' frbr_uri=work.commencing_work.frbr_uri %}" data-popup-url="{% url 'work_popup' frbr_uri=work.commencing_work.frbr_uri %}">{{ work.commencing_work.title }}</a>
                     {% endif %}
                   </dd>
+
+                  {% regroup work.taxonomies.all by taxonomy_vocabulary as topic_groups %}
+                  {% if topic_groups %}
+                      {% for topic in topic_groups %}
+                        <dt>{{ topic.grouper }}</dt>
+                        {% for taxonomy in topic.list %}
```
Comment (longhotsummer) [inline indigo_app/templates/indigo_api/work_overview.html]: If the `dd` spacing is poor, then perhaps wrap the entire for loop in one `dd` tag, and use `br` tags after each tox

**Comment by longhotsummer** [comment]
Example act with changes – Indigo Platform 2019-06-28 09-04-51

**Pull request merged**